libs/domain/picking/src/components/picking/picking.component.ts

```diff
@@ -150,20 +174,17 @@ export class PickingComponent extends PickingListMixin(LitElement) {
                     ></oryx-picking-product-card>
                   `
               )}
+              ${this.renderFinishButton()}
             `,
-            () => this.renderFallback()
+            () => html`${this.renderPlaceholderComplete()} `
           )}
         </div>`
       )}
     `;
   }
 
-  protected renderFallback(): TemplateResult {
-    if (
-      this.pickingList?.items.every(
-        (item) => item.status === ItemsFilters.Picked
-      )
-    ) {
+  protected renderPlaceholderComplete(): TemplateResult {
+    if (this.items?.every((item) => item.status === ItemsFilters.Picked)) {
       return html`
         <div class="picking-complete">
           <div class="img-wrap">
```

Comment:
```suggestion
  protected renderPlaceholderComplete(): TemplateResult | void {
    if (this.items?.some((item) => item.status !== ItemsFilters.Picked)) return
    
    return html`
      <div class="picking-complete">
        <div class="img-wrap">
```
And remove else statement